lib/topological_inventory/ansible_tower/targeted_refresh/service_instance.rb

```diff
@@ -55,6 +55,10 @@ def async_collecting_finished(entity_type, refresh_state_uuid, total_parts)
           logger.info("ServiceInstance#refresh: finished collecting of #{entity_type} (#{refresh_state_uuid}). Total parts: #{total_parts}")
         end
 
+        def response_received!
```

Comment:
good catch! What about changing the async_receiver to call it only if it reponds_to?

Reply:
I thought about doing it that way - but the class inherits from the AnsibleTower Collector class so we should probably have it implemented as a no-op rather than conditionally calling it in 4 different places.

Reply:
Ended up moving it into the base `AnsibleTower::Collector` class, that way the base class will catch it.